_security-analytics/api-tools/detector-api.md

```diff
@@ -7,60 +7,58 @@ nav_order: 35
+`triggers` | Array | Trigger settings for alerts.
+`triggers.ids` | Array | A list of rule IDs that become part of the trigger condition.
+`triggers.tags` | Array | Tags are specified in a security rule. Tags can then be selected and applied to the alert trigger to focus the trigger conditions for alerts. See an example of how tags are used in a Sigma rule in Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#tags).
+`triggers.id` | String | The unique ID for the trigger.
+`triggers.sev_levels` | Array | Sigma rule severity levels: `informational`; `low`; `medium`; `high`; `criticial`. See [Level](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#level) in the Sigma Rule Creation Guide.
+`triggers.name` | String | The name of the trigger. Name should only consist of upper and lowercase letters, numbers 0-9, hyphens, spaces, and underscores. Use between 5 and 50 characters. Required.
+`triggers.severity` | Integer | Severity level for the trigger expressed as an integer: 5 = lowest; 4 = low; 3 = medium; 2 = high; 1 = highest. Trigger severity is part of the alert definition.
+`triggers.actions` | Object | Actions send notifications when trigger conditions are met. Optional, as a notification message is not required as part of an alert.
+`triggers.actions.id` | String | Unique ID for the action. User generated.
+`triggers.actions.destination_id` | String | Unique ID for the notification destination. User generated. 
+`triggers.actions.subject_template` | Object | Contains the information for the subject field of the notification message. Optional.
+`triggers.actions.subject_template.source` | String | The subject for the notification message.
+`triggers.actions.subject_template.lang` | String | The scripting language used to define the subject. Must be Mustache. See the [Mustache Manual](https://mustache.github.io/mustache.5.html) for more information about templates.
+`triggers.actions.name` | String | Name for the trigger alert. Name should only consist of upper and lowercase letters, numbers 0-9, hyphens, spaces, and underscores. Use between 5 and 50 characters.
+`triggers.actions.message_template` | String | Contains the information for the body of the notification message. Optional.
+`triggers.actions.message_template.source` | String | The body of the notification message.
+`triggers.actions.message_template.lang` | String | The scripting language used to define the message. Must be `Mustache`.
+`triggers.actions.throttle_enabled` | Boolean | Enables throttling for alert notifications. Optional. Default is `false`.
+`triggers.actions.throttle` | Object | Throttling limits the number of notifications you receive within a given span of time.
+`triggers.actions.throttle.unit` | String | Unit of time for throttling.
+`triggers.actions.throttle.value` | Integer | The value for the unit of time.
```

Comment:
The value for throttling? It would be nice here to have a description of  how throttling happens? Is it that it sends an alert once in the time period specified by `value`?